include/phasar/PhasarPass/Options.h

```diff
@@ -15,25 +15,25 @@
 
 #include "llvm/Support/CommandLine.h"
 
-extern llvm::cl::OptionCategory PhASARCategory;
+extern llvm::cl::OptionCategory PhASARCategory; // NOLINT
```

Comment:
Are externs not allowed? If this should be so, maybe we should consider removing the clang-tidy rule?